components/thumbnail/basic/src/index.js

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+
+class ThumbnailBasic extends Component {
+  render () {
+    const { image, captionText } = this.props
+    return (
+      <figure className='sui-ThumbnailBasic'>
+        <img className='sui-ThumbnailBasic-Image' src={image.src} alt={image.alt} />
+        <figcaption className='sui-ThumbnailBasic-Caption'>
+          <span>{captionText}</span>
+        </figcaption>
+      </figure>
+    )
+  }
+}
+
+ThumbnailBasic.displayName = 'ThumbnailBasic'
+
+ThumbnailBasic.propTypes = {
+  image: PropTypes.object.isRequired,
+  captionText: PropTypes.string
```

Comment:
You'r right, after talking with Norbert, it makes no sense for us to use this component without caption so the caption will be required